test/ctr.bats

```diff
@@ -108,3 +108,36 @@ function teardown() {
 	cleanup_pods
 	stop_ocid
 }
+
+@test "ctr status #127" {
```

Comment:
I think that adding that info in commit message should be discoverable. Also, we could add comments in the code referring to an issue if it is handling some particularly hairy corner case. A description like "all ctrs in pod status" should work, IMO.